### PR TITLE
[HVG-839] Fix GraphQL API calls

### PR DESCRIPTION
**Pull request by sampettersson**
Seems like I implemented most of this earlier, this simply fixes so that API calls work again.

### CODE REVIEW

**Review by tarikstafford — APPROVED** [review]

File: Projects/Embark/Example/Sources/AppDelegate.swift [hunk 1]
```diff
@@ -75,20 +18,37 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         window = UIWindow(frame: UIScreen.main.bounds)
         window?.makeKeyAndVisible()
 
-        _ = ApolloClient.createClient(token: nil)
+        let apolloEnvironmentConfig = ApolloEnvironmentConfig(
+            endpointURL: URL(string: "https://graphql.dev.hedvigit.com/graphql")!,
+            wsEndpointURL: URL(string: "wss://graphql.dev.hedvigit.com/subscriptions")!,
+            assetsEndpointURL: URL(string: "https://graphql.dev.hedvigit.com")!
+        )
```
[the remark this answers is not in the document]
Comment (sampettersson) [reply Projects/Embark/Example/Sources/AppDelegate.swift]: Yeah, no it doesn't make much sense, the thing is that `hGraphQL` has no dependencies of it's own, so it's kind of isolated. Refactored a bit now so it's a bit cleaner tho.

File: Projects/Embark/Sources/EmbarkState+API.swift [hunk 2]
```diff
@@ -195,13 +195,18 @@ extension EmbarkState {
 
     private func handleApiRequest(apiFragment: GraphQL.ApiFragment) -> Future<ResultMap?> {
         func performHTTPCall(_ query: String, variables: ResultMap) -> Future<ResultMap?> {
-            var urlRequest = URLRequest(url: apolloEnvironment.endpointURL)
+            var urlRequest = URLRequest(url: ApolloClient.environment!.endpointURL)
```
[the remark this answers is not in the document]
Comment (sampettersson) [reply Projects/Embark/Sources/EmbarkState+API.swift]: For sure! Cleaned this up :)

File: Projects/Embark/Example/Sources/AppDelegate.swift (hunk 1 above)
[the remark this answers is not in the document]
Comment (tarikstafford) [reply Projects/Embark/Example/Sources/AppDelegate.swift]: `extension ApplicationState.Environment {
    var apolloEnvironmentConfig: ApolloEnvironmentConfig {
        switch self {
        case .staging:
            return ApolloEnvironmentConfig(
                endpointURL: URL(string: "https://graphql.dev.hedvigit.com/graphql")!,
                wsEndpointURL: URL(string: "wss://graphql.dev.hedvigit.com/subscriptions")!,
                assetsEndpointURL: URL(string: "https://graphql.dev.hedvigit.com")!
            )
        case .production:
            return ApolloEnvironmentConfig(
                endpointURL: URL(string: "https://giraffe.hedvig.com/graphql")!,
                wsEndpointURL: URL(string: "wss://giraffe.hedvig.com/subscriptions")!,
                assetsEndpointURL: URL(string: "https://giraffe.hedvig.com")!
            )
        case let .custom(endpointURL, wsEndpointURL, assetsEndpointURL):
            return ApolloEnvironmentConfig(
                endpointURL: endpointURL,
                wsEndpointURL: wsEndpointURL,
                assetsEndpointURL: assetsEndpointURL
            )
        }
    }
}
`

It's hard coded anyways isn't it?

[the remark this answers is not in the document]
Comment (sampettersson) [reply Projects/Embark/Example/Sources/AppDelegate.swift]: Yeah, but it's a part of `hCore` not `hGraphQL`, we gonna get a circular dependency otherwise 😅

**Review by tarikstafford — APPROVED** [review]

**Pull request merged**